app/models/early_career_payments.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# Module namespace specific to the policy for claiming early career payments.
+#
+# TODO: Put in details of what this is for.
+module EarlyCareerPayments
+  extend self
+
+  POLICY_START_YEAR = AcademicYear.new(2021).freeze
+
+  def start_page_url
+    if Rails.env.production?
+      "https://www.gov.uk/guidance/early-career-payments-guidance-for-teachers-and-schools"
+    else
+      "/#{routing_name}/claim"
+    end
+  end
+
+  def eligibility_page_url
+    "https://www.gov.uk/publications/TO-BE-REPLACED-by-response-to-ECP-518"
+  end
+
+  def routing_name
+    "early-career-payments"
+  end
+
+  def locale_key
+    routing_name.underscore
+  end
+
+  def notify_reply_to_id
+    "3f85a1f7-9400-4b48-9a31-eaa643d6b977"
+  end
+
+  def feedback_url
+    "https://docs.google.com/forms/TO-BE-REPLACED-by-response-to-ECP-509/viewform"
+  end
+
+  def short_name
+    I18n.t("early_career_payments.policy_short_name")
+  end
+
+  def first_eligible_qts_award_year(claim_year = nil)
+    POLICY_START_YEAR
+  end
+
+  def last_ineligible_qts_award_year
```

Comment:
No specs.

Reply:
@nomasprime While this is potentially poor form, not of the existing specs for M+P / TSLR modules have test coverage of this method. Given its a year less than the spec above for `first_eligible_qts_award_year`, I suspect that the delivering developers felt that this could slip